common/djangoapps/django_comment_common/tests.py

```diff
@@ -1,9 +1,9 @@
+from django.test import TestCase
 from nose.plugins.attrib import attr
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
 
-from django.test import TestCase
```

Comment:
It did, but I think I ran isort on this file back before I realized I didn't need to do anything in it. I can change it back since I'm not actually changing any code here?